### PR TITLE
[draft-js] Add a few `DraftEditor` member types

### PR DESCRIPTION
**Pull request by ruipserra**
### Changes

Add types for the following `DraftEditor` members:
- `getEditorKey`
- `editor`
- `editorContainer`

You can see the original flow types in the draft-js code here: https://github.com/facebook/draft-js/blob/d974abb0c2844bf99724aba9448df42a63291e0e/src/component/base/DraftEditor.react.js#L187-L189.

---

Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/facebook/draft-js/blob/d974abb0c2844bf99724aba9448df42a63291e0e/src/component/base/DraftEditor.react.js#L187-L189

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## draft-js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/draft-js@latest/))
was missing the following properties:
1. EntityInstance
1. RawDraftContentState
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a22f86e90e1d81e51be955c0db52d053f97e95c3
</p>

File: types/draft-js/index.d.ts [hunk 1]
```diff
@@ -53,6 +53,10 @@ declare namespace Draft {
              * state of the editor. See `DraftEditorProps` for details.
              */
             class DraftEditor extends React.Component<DraftEditorProps, {}> {
+                editor: HTMLElement | null | undefined;
+                editorContainer: HTMLElement | null | undefined;
```
Comment (martin-badin) [inline types/draft-js/index.d.ts:57]: Please update

```suggestion
                editor?: HTMLElement | null | undefined;
                editorContainer?: HTMLElement | null | undefined;
```

**Review by martin-badin — CHANGES_REQUESTED** [review]

**Review by martin-badin — CHANGES_REQUESTED** [review]
Please, could you add tests?

File: types/draft-js/index.d.ts [hunk 2]
```diff
@@ -53,6 +53,10 @@ declare namespace Draft {
...
+                getEditorKey: () => string;
```
Comment (martin-badin) [inline types/draft-js/index.d.ts:58]:
```suggestion
                getEditorKey(): string;
```

**Review by martin-badin — CHANGES_REQUESTED** [review]

**Review by martin-badin — APPROVED** [review]

**Review by amcasey — APPROVED** [review]

**Comment by ruipserra** [comment]
Ready to merge

**Pull request merged**